src/components/CollectionBar/CollectionBar.tsx

```diff
@@ -73,7 +73,7 @@ export const CollectionBar = ({
             <Button
               variant='tertiary'
               size='small'
-              color={color === 'dark' ? 'inverted' : undefined}
+              className={cn(classes.buttonText, classes[color])}
```

Comment:
`inverted` color for \<Button\> is deprecated. Use new css class instead to set text color.